web/src/osm2streets-svelte/Layout.svelte

```diff
@@ -0,0 +1,23 @@
+<div class="left">
+  <slot name="left" />
+</div>
+<div class="main">
+  <slot name="main" />
+</div>
+
+<style>
+  div {
+    float: left;
+    padding: 8px;
+  }
+
+  .left {
+    width: 25%;
+    height: 100vh;
+    overflow: scroll;
+  }
+  .main {
+    /* TODO 75% wraps?! */
```

Comment:
Even 73% is broken on my other laptop with a 2x display factor. There's some basic CSS stuff I'm still not understanding. :\

Reply:
It's to do with the [`box-sizing` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing): by default, when you write `width: 73%` this refers to how wide the _contents_ of the div are. The 8px padding is then added on top of that, so that the entire div is actually 73% + 8px wide (very counterintuitive...)

The solution is to add `box-sizing: border-box` to both `div.left` and `div.main`. Or you can go one step further and [apply this globally]((https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/)), so that you don't run into this problem elsewhere. That's what [I'm doing now](https://github.com/alan-turing-institute/demoland-web/blob/main/src/main.css), at least. 😄 

```css
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: inherit;
}
```

Reply:
Ahh that's indeed an odd default (legacy choice, if I had to guess). Thanks for the tip + explanation! I've fixed just in Layout for now, because I'm not exactly sure about osm2streets-svelte (intended as an NPM library) overriding a global style, or if every app using it should generally do that.